src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALLS_COUNT = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < BALLS_COUNT; ++i) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```

There is no need to explicitly call toString on the objects we pass to the println method. There it is called under the hood